packages/jest-editor-support/src/__tests__/fixtures/failing_jsons/failing_jest_json.json

```diff
@@ -0,0 +1,214 @@
eueRunner.run (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1939:9)\n    at /Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1966:16\n    at /Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1909:9\n    at Object.fn (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/build/jasmine-async.js:68:11)\n    at attemptAsync (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1984:24)\n    at QueueRunner.run (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1939:9)\n    at QueueRunner.execute (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1927:10)\n    at Spec.queueRunnerFactory (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:718:35)\n    at Spec.execute (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:372:10)\n    at Object.fn (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:2584:37)\n    at attemptAsync (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1984:24)\n    at QueueRunner.run (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1939:9)\n    at QueueRunner.execute (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1927:10)\n    at queueRunnerFactory (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:718:35)\n    at Object.fn (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:2569:13)\n    at attemptAsync (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1984:24)\n    at QueueRunner.run (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1939:9)\n    at QueueRunner.execute (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:1927:10)\n    at queueRunnerFactory (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:718:35)\n    at TreeProcessor.execute (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:2431:7)\n    at Env.execute (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/vendor/jasmine-2.5.2.js:780:17)\n    at jasmine2 (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-jasmine2/build/index.js:96:7)\n    at runTest (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-cli/build/runTest.js:50:10)\n    at promise.then (/Users/orta/dev/projects/danger/danger-js/node_modules/jest-cli/build/TestRunner.js:304:14)\n    at process._tickCallback (internal/process/next_tick.js:103:7)"
```

Comment:
Yep, everything internally uses absolute paths, and so for the fixtures to be realistic, I kept the paths in